asyncapi/WebRTC/signaling.yaml

```diff
@@ -0,0 +1,1296 @@
+        - $ref: '#/components/messageTraits/correlationHeader'
+      payload:
+        allOf:
+          - $ref: '#/components/schemas/commonPayload'
+          - $ref: '#/components/schemas/requestType'
+          - $ref: '#/components/schemas/requestPayload'
+          - $ref: '#/components/schemas/initSessionMethod'
+          - $ref: '#/components/schemas/initSessionParams'
+      x-response:
+        oneOf:
+          - $ref: '#/components/messages/initSessionResponse'
+          - $ref: '#/components/messages/initSessionAck'
+          - $ref: '#/components/messages/errorFromServer'
+    initSessionResponse:
+      description: >-
+        This message is sent from the signaling server to the client in response
+        to an initSession request. It echoes back the initSession message but
+        adds details for the STUN and TURN servers that can be used. The payload
+        contains the exact same data that was sent in the request message.
+      correlationId:
+        location: $message.payload#/context
+      traits:
+        - $ref: '#/components/messageTraits/commonHeader'
+        - $ref: '#/components/messageTraits/initSessionType'
+        - $ref: '#/components/messageTraits/correlationHeader'
+        - $ref: '#/components/messageTraits/serverHeader'
+      payload:
+        allOf:
+          - $ref: '#/components/schemas/commonPayload'
+          - $ref: '#/components/schemas/requestType'
+          - $ref: '#/components/schemas/requestPayload'
+          - $ref: '#/components/schemas/initSessionMethod'
+          - $ref: '#/components/schemas/initSessionParams'
+    initSessionAck:
+      description: >-
+        The device sends this message as an acknowledgement that it has received
+        an initSession message.
+      correlationId:
+        location: $message.payload#/context
+      traits:
+        - $ref: '#/components/messageTraits/commonHeader'
+        - $ref: '#/components/messageTraits/signalingType'
+      payload:
+        allOf:
+          - $ref: '#/components/schemas/commonPayload'
+          - $ref: '#/components/schemas/responseType'
+          - $ref: '#/components/schemas/requestPayload'
+          - $ref: '#/components/schemas/initSessionMethod'
+          - $ref: '#/components/schemas/ackPayload'
+      examples:
+        - summary: Example acknowledgement after successful validation.
+          payload:
+            apiVersion: '1.0'
+            type: response
+            sessionId: 01GMSYPBS4V3KBJYVYT0HA46RX
+            method: initSession
+            context: 01GMSYHPEEKC4YCQ784N0ZB4DQ
+            data: {}
+        - summary: Example error response.
+          payload:
+            apiVersion: '1.0'
+            type: response
+            sessionId: 01GMSYPBS4V3KBJYVYT0HA46RX
+            method: initSession
+            context: 01GMSYHPEEKC4YCQ784N0ZB4DQ
+            error:
+              code: 2103
+              message: 'Required parameter(s) missing: ''sessionId'''
+    initSession2:
```

Comment:
Posted here but applies to all the `2`s. Since this name is pretty much only present for documentation purposes (it doesn't show up in the messages) I would go with something more verbose. I feel it would be better with "DeviceInitSession" and "ClientInitSession" to be clear which side of the API it belongs to.